spec/system/task_spec.rb

```diff
@@ -43,11 +45,10 @@
   end
 
   describe 'Task詳細' do
+    let!(:task) { create(:task, project_id: project.id)}
```

Comment:
Factoryに`association`を設定すると`project_id`の指定をしなくて済みます。
```suggestion
    let!(:task) { create(:task)}
```

Reply:
修正しました

Reply:
[b47f561](https://github.com/MasayaShimayori/rspec_app_exam/pull/1/commits/b47f561ee6297f8ac97e97e3fc1b84bbfb726d26)
こちらで修正しました

Reply:
[70b1075](https://github.com/MasayaShimayori/rspec_app_exam/pull/1/commits/70b1075561bd51755d09afd0ebee28a1500f81d0)
こちらで修正しました。